firstpartydata/first_party_data.go

```diff
@@ -656,3 +690,37 @@ func ExtractFPDForBidders(req *openrtb_ext.RequestWrapper) (map[openrtb_ext.Bidd
 	return ResolveFPD(req.BidRequest, fbdBidderConfigData, globalFpd, openRtbGlobalFPD, biddersWithGlobalFPD)
 
 }
+
+func mergeContents(originalContent *openrtb2.Content, fpdBidderConfigContent json.RawMessage, globalContentData []openrtb2.Data) (*openrtb2.Content, error) {
+	if originalContent == nil {
```

Comment:
No, not really. It means we don't need to run any merging functionality if `originalContent == nil`. 
If `openRtbGlobalFPD` is present it will be handled upstream after `mergeSites` or `mergeApps` function in `resolveSite` and `resolveApp`.

Reply:
I also added 2 new tests to cover this cases